darwin/torch/dataset.py

```diff
@@ -389,14 +389,12 @@ def _map_annotation(self, index: int):
 
         target = []
         for annotation in annotations:
-            assert "name" in annotation
-            assert "polygon" in annotation
-            # Extract the sequences of coordinates from the polygon annotation
-            sequences = convert_polygons_to_sequences(annotation["polygon"]["path"])
-            # Discard polygons with less than three points
-            sequences[:] = [s for s in sequences if len(s) >= 6]
-            if not sequences:
+            if "polygon" not in annotation and "complex_polygon" not in annotation:
+                print(f"Warning: missing polygon in annotation {self.annotations_path[index]}")
```

Comment:
I don't think assertions belong here tbh, I'd much rather print a warning and continue.
Let me know if you agree